.github/workflows/test.yml

```diff
@@ -212,6 +212,5 @@ jobs:
 
     cross-platform:
         name: Cross-platform tests
-        if: github.event_name != 'pull_request'
```

Comment:
why did we use this?

Reply:
Maybe to not get spammed? I removed it just so we can test the CI here in this pr